core/domain/suggestion_services.py

```diff
@@ -2965,6 +3436,221 @@ def increment_question_review_stats(
         last_contribution_date.date())
 
 
+def increment_translation_submitter_total_stats_at_review(
+    translation_submitter_total_stat: (
+        suggestion_registry.TranslationSubmitterTotalContributionStats),
+    content_word_count: int,
+    suggestion_is_accepted: bool,
+    edited_by_reviewer: bool
+) -> None:
+    """Updates TranslationSubmitterTotalContributionStats object.
+
+    Args:
+        translation_submitter_total_stat:
+            TranslationSubmitterTotalContributionStats. The stats object to
+            update.
+        content_word_count: int. The number of words in the translation.
+        suggestion_is_accepted: bool. A flag that indicates whether the
+            suggestion is accepted.
+        edited_by_reviewer: bool. A flag that indicates whether the suggestion
+            is edited by the reviewer.
+    """
+    # Weights for calculating performance.
+    # recent_performance = accepted cards - 2 (rejected cards) in last
+    # 100 contributions.
+    if suggestion_is_accepted:
+        translation_submitter_total_stat.accepted_translations_count += 1
+        translation_submitter_total_stat.accepted_translation_word_count += (
+            content_word_count)
+        translation_submitter_total_stat.overall_accuracy = round((
+            translation_submitter_total_stat.accepted_translations_count
+            / translation_submitter_total_stat.submitted_translations_count
+        ), 3) * 100
+
+        if (
+                len(translation_submitter_total_stat
+                    .recent_review_outcomes)
+                > 99
```

Comment:
Why 99? Can this be a constant somewhere?

Reply:
Yeah that sounds good, Updated.